tests/Versioning/VersionHistoryTest.php

```diff
@@ -311,6 +313,119 @@ public function testDeleteUnexistingVersion()
     }
 
     /**
+     * Load Version by label
+     */
```

Comment:
maybe add `@depends testAddLabel`? if adding the label does not work, this will fail for sure.